### ISSUE TITLE
Add support for name attribute

### ISSUE THREAD

**Issue by xgvargas** [pycollada/pycollada#55]
I'm trying to read a `.dae` file exported from sketchup and I'm having trouble to get the name of each geometry from the file. The `id` property is fine, but the `name` is always blank.

``` xml
<library_nodes>
    <node id="ID4" name="i_am_the_block_name">
        <instance_geometry url="#ID5">
            </bind_material>
                <!-- ...... -->
            </bind_material>
        </instance_geometry>
    </node>
    <!-- ...... -->
</library_nodes>
```

How can I have access to this property?

**Comment by jterrace** [comment]
I believe the library currently doesn't parse name at all.

**Comment by xgvargas** [comment]
That was fast!!

It's an important part of my project. I'll try to add this feature by myself. Any tip?

**Comment by jterrace** [comment]
Actually, it should be pretty easy to add support for it. You can see we set the name=id when saving the node here:
https://github.com/pycollada/pycollada/blob/master/collada/scene.py#L389

The load function should pull out the name attribute and pass it to the constructor. I'd be happy to review a pull request. Thanks!

**Comment by jterrace** [comment]
Fixed in #131
